ROADMAP.md

```diff
@@ -12,7 +12,7 @@ The goal of the .NET for Apache Spark project is to provide an easy to use, .NET
 ### Performance Optimizations
 * Improvements to C# Pickling Library
 * Improvements to Arrow .NET Library
-* Exploiting .NET Core 3.0 Vectorization (*)
+* Exploiting .NET 6 Vectorization (*)
```

Comment:
I don't know, but I suspect this find-replace change isn't desired.

Reply:
Make it a version-independent statement? "Exploit .NET SIMD intrinsics?"

Reply:
@AFFogarty yes let's remove this `6`

Reply:
Removed